demos/common/python/models/yolo.py

```diff
@@ -54,30 +54,22 @@ def __init__(self, param, sides):
 
                 self.isYoloV3 = True  # Weak way to determine but the only one.
 
-    def __init__(self, ie, model_path, labels=None, keep_aspect_ratio=False, threshold=0.5, iou_threshold=0.5):
-        super().__init__(ie, model_path)
-
+    def __init__(self, ie, model_path, input_transform=None, resize_type='default', 
+                 labels=None, threshold=0.5, iou_threshold=0.5):
+        super().__init__(ie, model_path, input_transform=input_transform, resize_type=resize_type, 
+                         labels=labels, threshold=threshold, iou_threshold=iou_threshold)
         self.is_tiny = self.net.name.lower().find('tiny') != -1  # Weak way to distinguish between YOLOv4 and YOLOv4-tiny
```

Comment:
we were going to check number of outputs instead of name, right ?

Reply:
Dont worry, the YOLO wrappers will be refactored as well

Reply:
The issue is still there. Is it going to be fixed in separate PR ?